### PR TITLE
[EASI-3691] Geography questions updates

### PR DESCRIPTION
**Pull request by OddTomBrooks**
# EASI-3691

## Changes and Description

- Frontend
  - Updated General Characteristics read only section 
  - Update `TargetsAndOPtions` forms 
  - Add new `geographiesStatesAndTerritories` and `geographiesRegionTypes` to apolloGQL files
  - Add new `geographiesStatesAndTerritories` and `geographiesRegionTypes` translation files
- Backend
  - Added enums
  - Added column for geographies region type
  - Added column for geographies states and territories type
  - Updated schema and DTOs

### Frontend
1. Go to Page 4 in General Characteristics
2. Fill out Targeted Geography question
3. Ensure data persist, is in csv, and read only

### Backend
- Bring server containers up
- Test endpoints
- Assert no errors and database values updated accordingly

## PR Author Review Checklist

- [x] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [x] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [x] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]
Thanks for opening the PR @OddTomBrooks, Can you please pull in main and address merge conflicts?

**Review by patrickseguraoddball — DISMISSED** [review]
LG

File: MINT.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
 	"info": {
+<<<<<<< HEAD
```
Comment (StevenWadeOddball) [inline MINT.postman_collection.json:3]: I think this file was missed when resolving merge conflicts

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]

**Comment by StevenWadeOddball** [comment]
Functionally this still looks good! I just found some merge conflict notation in the postman file. Can you correct that please?

**Review by StevenWadeOddball — APPROVED** [review]
Thanks Tom, this looks good! It looks like there might be some snapshots that need to be updated that will prevent merging.

**Comment by OddTomBrooks** [comment]
@patrickseguraoddball There is a merge conflict in one of the FE tests, may be better for you or @garyjzhao to tackle this one

**Comment by patrickseguraoddball** [comment]
> @patrickseguraoddball There is a merge conflict in one of the FE tests, may be better for you or @garyjzhao to tackle this one

I'll handle it

**Review by StevenWadeOddball — DISMISSED** [review]

**Review by StevenWadeOddball — APPROVED** [review]

**Pull request merged**